tests/test_file_processing.py

```diff
@@ -1571,7 +1572,7 @@ def test_insert_plates_and_wells_from_docs_into_dart_multiple_new_plates(config)
             FIELD_PLATE_BARCODE: "TC-rna-00000029",
             FIELD_COORDINATE: "A01",
             FIELD_LAB_ID: "AP",
-            FIELD_RESULT: POSITIVE_RESULT_VALUE,
+            FIELD_RESULT: generate_new_object_for_string(POSITIVE_RESULT_VALUE),
```

Comment:
I think it would be more readable to use "Positive" than this function - would this achieve the same result?

Reply:
We tried that, but it wasn't enough to trick Python so it does not re-use the same reference to the string 'Positive' from somewhere else. We also tried: 
```
'Posi'+'tive'
```
but it didnt work either, because python was clever enough to replace this string at pre-execution time with the previous reference of 'Positive' in memory, and that passed the tests in the previous version of code.

Reply:
Interesting!